fits/FITS/fits.h

```diff
@@ -91,15 +91,14 @@ class FitsLogical {
 	friend ostream & operator << (ostream &o, const FitsLogical &);
     public:
 	FitsLogical() : v('\0') { }
-	FitsLogical(Bool x) { v = (x == True ? 'T' : 'F'); }
-	FitsLogical(const FitsLogical &x) : v(x.v) { }
-	FitsLogical & operator = (const FitsLogical &x) { 
```

Comment:
Quite a lot of functions disappear here, why?

Reply:
For copy constructors, they are redundant: the default copy constructor is better. A manual copy constructor disables the assignment operator (or a move constructor), although gcc so far accepted us of an assingment operator like that. Defining only one of a copy constructor or copy assignment operator causes a warning when the other one is used.

Reply:
Thanks, clear.